File: Aufgabennstellung 6/Haskell/CountEven.hs
main :: IO ()
main = putStrLn $
    let numbers = [1,2,4,8,9,10,14,6,3,5,7,9,13,14,15,16,17,18,200,101,211,122]
        evenNumbers = filter even numbers
        amount = length evenNumbers
    in "Liste: " ++ show numbers ++ "\nAnzahl gerader Zahlen: " ++ show amount

File: Aufgabennstellung 6/Python/count_even.py
def count_even(numbers):
    counter = 0
    for number in numbers:
        if (number % 2) == 0:
            counter += 1

    return counter


numberList = [1,2,4,8,9,10,14,6,3,5,7,9,13,14,15,16,17,18,200,101,211,122]

print(f"Anzahl gerader Zahlen: {count_even(numberList)}")
